Week1/queries.js

```diff
@@ -0,0 +1,98 @@
+const mysql = require('mysql');
+const connection = mysql.createConnection({
+  host: 'localhost',
+  user: 'hyfuser',
+  password: 'hyfpassword',
+  database: 'world',
+});
+
+connection.connect(error => {
+  if (error) {
+    throw error;
+  }
+  console.log('Connected.');
+});
+
+const queries = {
+  createDatabase: `CREATE DATABASE IF NOT EXISTS world;`,
+
+  useDatabase: `USE worldData`,
+
+  createTableCountry: `CREATE TABLE IF NOT EXISTS country (code CHAR(3), Name CHAR(52), Continent CHAR (30), Region CHAR(26),SurfaceArea FLOAT(10,2),Indepyear SMALLINT(6),Population INT(11),LifeExpectancy FLOAT(3.1),GNP FLOAT(10.2),LocalName CHAR(45),GovernmentForm CHAR(45),HeadOfState CHAR(60),Capital INT(11),Code2 CHAR(2),PRIMARY KEY(Code));`,
+
+  createTableCity: `CREATE TABLE IF NOT EXISTS city(ID INT(11) AUTO_INCREMENT,Name CHAR(35),CountryCode CHAR(3),District CHAR(20),Population INT(11),PRIMARY KEY (ID),KEY CountryCode (CountryCode))`,
+
+  useDatabaseTwo: `USE world`,
+};
+
+Object.keys(queries).forEach(function(item) {
+  connection.query(queries[item], error => {
+    if (error) {
+      throw error;
+    }
+    console.log('Query successful...');
+  });
+});
+
+//Queries
+// i.
+const country_pop8mil = `SELECT * FROM country WHERE Population > 8000000`;
```

Comment:
using `select *` in real system is not recommended. Always only retrieve things that you actually intend to use https://stackoverflow.com/questions/3639861/why-is-select-considered-harmful

Reply:
I read it and understood. But can you give me an example of it in my queries? For example for the first query.. How can I do that instead of using select * ?

Reply:
Okai, I think I got that. I committed updated version with the changes of 'select *'.